src/python/pants/backend/python/dependency_inference/module_mapper_test.py

```diff
@@ -393,14 +410,20 @@ def req(
             req("multiple_owners_types", "types-multiple_owners==1", resolve="another"),
             # Only assume it's a type stubs dep if we are certain it's not an implementation.
             req("looks_like_stubs", "looks-like-stubs-types", modules=["looks_like_stubs"]),
+            req("google-cloud-hardyhar", "google-cloud-hardyhar"),
```

Comment:
**comment:** added a sample of mappings from the original list
we could add all of the mappings from the original list to feel better about the changes, but that would be a large list. instead, i added the minimum representation of the different edge cases i thought of.